master/custom/workers.py

```diff
@@ -281,4 +281,10 @@ def get_workers(settings):
             parallel_tests=2,
             parallel_builders=2,
         ),
+        cpw(
+            name="ambv-bb-win11",
+            tags=['windows', 'win11', 'amd64', 'x86-64', 'bigmem'],
+            branches=['3.9', '3.10','3.11', '3.x'],
```

Comment:
```suggestion
            branches=['3.9', '3.10', '3.11', '3.x'],
```

Reply:
Actually, come to think, you can leave this out.  The default is all current branches.